_posts/2020-11-27-eosc-nordic-article.md

```diff
@@ -0,0 +1,12 @@
+---
+site: freiburg
+title: Insights from the first cross-training between EOSC-Life and EOSC-Nordic
+tags:
+- training
+- press
+supporters:
+- unifreiburg
+- elixir
+- eosc
+external: https://www.eosc-nordic.eu/insights-from-the-first-cross-training-between-eosc-life-and-eosc-nordic/
+---
```

Comment:
If we add a small sentence to this post, it will be shown on the summary page. This usually looks better than without any content.